### DISCUSSION TITLE
How to create this kind of banner img?

### DISCUSSION THREAD

**Discussion by AuroraTea** [BEPb/BEPb#37] [General]
https://github.com/BEPb/BEPb/blob/main/src/header_.png

**Comment by Sudhanshu-Ambastha** [comment]
I recommend utilizing the [GitHub Profile Header Generator website](https://leviarista.github.io/github-profile-header-generator/) available at [GitHub Profile Header Generator source code](https://github.com/leviarista/github-profile-header-generator). It is an effective tool for creating professional banners or you can use this one also [Figma GitHub banner generator ](https://www.figma.com/community/file/1206877665795271691/readme-github-banner)

**Comment by BEPb** [comment]
no need to reinvent the wheel, everything is already written in the .md file
